linux_os/guide/system/auditing/auditd_configure_rules/audit_unsuccessful_file_modification/audit_rules_unsuccessful_file_modification_open_o_creat/oval/shared.xml

```diff
@@ -0,0 +1,200 @@
+<def-group>
+  <definition class="compliance" id="audit_rules_unsuccessful_file_modification_open_o_creat" version="1">
+    <metadata>
+      <title>Ensure auditd Collects Information on Unsuccesful Creation Attemps to Files</title>
```

Comment:
Ensure auditd Collects Information on Unsuccesful Creation Attempts to Files - o_creat